starter-code/ShoppingListVer2/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistver2/GroceryItem.java

```diff
@@ -0,0 +1,31 @@
+package ly.generalassemb.drewmahrt.shoppinglistver2;
+
+/**
+ * Created by Joe on 10/25/16.
+ */
+
+public class GroceryItem {
+    private String mName;
+    private String mDescription;
+
+    public GroceryItem(String name, String description) {
+        mName = name;
+        mDescription = description;
+    }
+
+    public String getDescription() {
+        return mDescription;
+    }
+
+    public void setDescription(String description) {
+        mDescription = description;
+    }
+
+    public String getName() {
+        return mName;
+    }
+
+    public void setName(String name) {
+        mName = name;
```

Comment:
Totally fine to have setters here, but I just want to make sure you know you don't always need to have setters. Sometimes you don't want an object's values to change after it's created, so you might prefer not to provide any setters.